crates/oxc_ast/src/ast/js.rs

```diff
@@ -535,18 +579,17 @@ pub struct ArrayExpression<'a> {
 }
 
 inherit_variants! {
-/// Array Expression Element
-///
-/// Inherits variants from [`Expression`]. See [`ast` module docs] for explanation of inheritance.
```

Comment:
Could you possibly restore these comments about inheritance? I know they're repeated throughout this file, but it's quite confusing, so I think best to err on side of making sure a reader sees the comment and has a chance of understanding what's going on!

Reply:
This change has been made.

Reply:
This has now been addressed.